### DISCUSSION TITLE
Updating `speedify.irb`

### DISCUSSION THREAD

**Discussion by jasikpark** [Homebrew/discussions#3221] [Tap maintenance and Homebrew development]
I recently made a PR that I thought would allow keeping the cask speedify up to date, https://github.com/Homebrew/homebrew-cask/pull/120920

but when I run `brew bump speedify` I get this output:

```
❯ brew bump speedify
==> speedify
Current cask version   :  11.9.2,8960
Latest livecheck version: 11.9.2,8960
Latest Repology version:  12.0.0.10367
Open pull requests:       none
```

Though I get "update to 12.0.0" reminders when I open the app / can manually download and update the app

**Comment by SMillerDev** [comment]
I think the problem is in their caching because my `brew livecheck` downloads a version with:
```xml
   <key>CFBundleShortVersionString</key>
   <string>11.9.2</string>
```

and my own download had:
```xml
    <key>CFBundleShortVersionString</key>
    <string>12.0.0</string>
```

**Reply by jasikpark** [reply]
Hmm weird.

**Reply by jasikpark** [reply]
Is there something I can do to fix this? Disable caching in the livecheck?

**Reply by jasikpark** [reply]
thanks for the response @SMillerDev :)

**Reply by SMillerDev** [reply]
I don't think so, caching would be on their end

**Reply by jasikpark** [reply]
Hmm would you mind running `brew bump speedify` yourself then? I'm still running into this issue..

should I update the URL used for livecheck in this case

**Reply by jasikpark** [reply]
I've added https://github.com/Homebrew/homebrew-cask/pull/123906 to use a url recommended by someone who works on the tool - hopefully it doesn't have the same caching problems
